platform/commonUI/general/src/controllers/ObjectInspectorController.js

```diff
@@ -38,6 +38,8 @@ define(
             $scope.contextutalParents = [];
             //$scope.isLink = false;
 
+            this.InspectorMutation = $scope.ngModel.selectedObject.getCapability('mutation');
```

Comment:
Variable naming is inconsistent here; first letter should be [lower-case](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md#code-guidelines)